src/main/java/com/dataloom/organizations/controllers/OrganizationsController.java

```diff
@@ -81,13 +86,14 @@ public Void destroyOrganization( @PathVariable( ID ) UUID organizationId ) {
         List<UUID> aclKey = ensureOwner( organizationId );
         authorizations.deletePermissions( aclKey );
         organizations.destroyOrganization( organizationId );
+        securableObjectTypes.deleteSecurableObjectType( ImmutableList.of( organizationId ) );
         return null;
     }
 
     @Override
     @PutMapping(
         value = ID_PATH + TITLE,
-        consumes = MediaType.TEXT_PLAIN_VALUE )
+        consumes = MediaType.APPLICATION_JSON_VALUE )
```

Comment:
This should be TEXT_PLAIN_VALUE. Something must be wrong elsewhere if you are getting a null request body. @UnsungHero97 Could loom-data-js be sending it with the wrong content type header or wrong encoding? Content type header probably correct or being ignored since 400 isn't thrown. Probably bad encoding.